src/lib/components/checklist.svelte

```diff
@@ -43,12 +43,10 @@
 	// };
 
 	function scrollToTop(event) {
-    event.preventDefault();
-    const mainElement = document.getElementById('main');
-    mainElement.scrollIntoView({ behavior: 'smooth' });
-  }
-
-  console.log(richtlijnen)
+		event.preventDefault();
```

Comment:
Zet event.preventDefault() altijd onderaan de functie.. mocht er iets misgaan in de client side js, dan kan de server het nog overnemen.

Reply:
✅